plugins/mongodb/examples/mongodb-test.ecl

```diff
@@ -0,0 +1,188 @@
+Import MongoDB;
+
+server := 'cluster0.nvkdfj5.mongodb.net';
+
+user := 'username';
+
+pwd := 'password';
+
+databaseName := 'mydb';
+
+collectionName := 'test1';
+
+// Records for defining the layout of example datasets
+reviewsRec := RECORD
+    INTEGER review_scores_cleanliness; 
+    INTEGER review_scores_checkin; 
+    INTEGER review_scores_communication; 
+    INTEGER review_scores_location; 
+    INTEGER review_scores_value; 
+    INTEGER review_scores_rating; 
+    INTEGER review_scores_accuracy;
+END;
+
+layoutairbnb := RECORD
+    UTF8 name; 
+    UTF8 space; 
+    UTF8 description; 
+    INTEGER beds; 
+    INTEGER accommodates;
+    SET OF STRING amenities; 
+    DATASET(reviewsRec) review_scores;
+END;
+
+transactionrec := {
+    INTEGER amount,
+    STRING transaction_code,
+    STRING symbol,
+    STRING price,
+    STRING total
+};
+
+layouttransactions := RECORD
+    INTEGER account_id;
+    INTEGER transaction_count;
+    DATASET(transactionrec) transactions;
+END;
+
+layoutFees := RECORD
+    DECIMAL32_2 price;
+    DECIMAL32_2 security_deposit;
+    DECIMAL32_2 cleaning_fee;
+    DECIMAL32_2 extra_people;
+    REAL guests_included;
+END;
+
+layoutDates := {STRING bucket_start_date, STRING bucket_end_date};
+layoutEmployee := {INTEGER1 id, STRING25 first, STRING25 last, REAL salary};
+layoutUpdate := {Integer matched_count, Integer modified_count};
+layoutperson := {String username, String address, String email};
+
+// Example/Test functions
```

Comment:
Please add a one-line comment to each example, roughly describing what each is supposed to be doing.